hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/fairness/AbstractRouterRpcFairnessPolicyController.java

```diff
@@ -63,10 +63,14 @@ public void init(Configuration conf) {
   @Override
   public boolean acquirePermit(String nsId) {
     try {
-      LOG.debug("Taking lock for nameservice {}", nsId);
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("Taking lock for nameservice {}", nsId);
```

Comment:
We have gone through this in the past.
LOG.debug with {} should already be equivalent to adding the safeguard.

Do you have some instrumentation showing that `LOG.isDebugEnabled()` is cheaper than just calling `LOG.debug()`?

Reply:
i.m sorry, @goiri ,  here does not need if condition statement exactlly. i will close this pr.